cmd/common/config/flags.go

```diff
@@ -23,6 +23,8 @@ const (
 	KeyTLSClientCertFile = "tls-client-cert-file" // string
 	KeyTLSClientKeyFile  = "tls-client-key-file"  // string
 	KeyTLSServerName     = "tls-server-name"      // string
+	KeyBasicAuthUser     = "basic-auth-user"      // string
```

Comment:
nit: I slightly prefer to spell out username fully, but not feeling too strongly about that one

```suggestion
	KeyBasicAuthUser     = "basic-auth-username"      // string
```